apps/sms/style/sms.css

```diff
@@ -32,6 +32,23 @@ form[role="dialog"][data-type="confirm"] p {
   white-space: pre-line;
 }
 
+form[role="dialog"].error-dialog-show-recipient ul {
+  border: none;
+}
+
+form[role="dialog"].error-dialog-show-recipient ul li {
+  padding: 0;
+}
+
+.error-dialog-show-recipient li {
```

Comment:
I see ...  maybe you can add a commet about the override part, or merge `.error-dialog-show-recipient li` into `form[role="dialog"].error-dialog-show-recipient ul li` with comment above the padding.